src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/simulering/SimuleringUtil.kt

```diff
@@ -72,11 +84,13 @@ fun fagområdeKoderForPosteringer(stønadType: StønadType): Set<FagOmrådeKode>
         FagOmrådeKode.ENSLIG_FORSØRGER_OVERGANGSSTØNAD_INFOTRYGD,
         FagOmrådeKode.ENSLIG_FORSØRGER_OVERGANGSSTØNAD_MANUELL_POSTERING,
     )
+
```

Comment:
Nå som du har endret MANUELL_POSTERING til MANUELL_POSTERING_INFOTRYGD så er jeg usikker på om det har noe å si for `fagområdeKoderForPosteringer` som nå ikke håndterer infotrygdposteringer lenger.. 🤔

Reply:
Endret

Reply:
Kanskje legge til de to nye og ?

Reply:
Ja, det er nok lurt